presto-tests/src/test/java/com/facebook/presto/tests/TestTpchDistributedQueries.java

```diff
@@ -23,4 +29,93 @@ public TestTpchDistributedQueries()
     {
         super(createQueryRunner());
     }
+
+    @Test
+    public void testExplainAnalyze()
```

Comment:
I prefer not to focus on testing for this patch. This is because @cberner still develops his version of explain and probably has vision on testing too. The reason why tests are hard is that stats are not collected in a consistent way (e.g. expressions don't always have predefined order, so regex matching is not enough). If you read the code, for some tests I have matching regex and for some I don't. The proper testing itself seem like a lot of work. I don't think it's worth it on our custom explain fork, which will likely be replaced with @cberner version.